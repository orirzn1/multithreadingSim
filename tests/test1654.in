L13
S8
O15
N10

T0
I@0x00000000
SUBI $1, $7, 0x2
SUBI $6, $4, 0x4
ADDI $4, $4, 0x1
SUBI $5, $5, 0x1
SUBI $2, $6, 0x4
LOAD $5, $5, $3
LOAD $2, $1, $7
SUB $3, $1, $4
ADDI $4, $6, 0x3
SUBI $5, $3, 0x2
HALT $0

T1
I@0x00000000
LOAD $1, $3, 0x4
SUB $6, $2, $4
SUB $3, $2, $4
SUBI $0, $7, 0x4
STORE $4, $6, 0x3
SUBI $2, $5, 0x1
HALT $0

T2
I@0x00000000
STORE $5, $3, $0
SUB $2, $0, $5
SUB $1, $0, $6
ADDI $7, $6, 0x4
HALT $0

T3
I@0x00000000
ADDI $2, $4, 0x1
STORE $7, $5, $7
LOAD $6, $0, $4
ADD $0, $5, $1
HALT $0

T4
I@0x00000000
ADDI $0, $6, 0x3
ADD $6, $3, $7
SUBI $5, $5, 0x2
ADD $6, $7, $6
STORE $4, $5, 0x2
ADD $7, $0, $0
HALT $0

T5
I@0x00000000
LOAD $3, $1, 0x1
SUB $1, $6, $2
STORE $7, $1, 0x4
LOAD $3, $6, $2
ADDI $6, $6, 0x1
STORE $1, $2, $1
LOAD $6, $3, 0x3
ADD $3, $6, $7
HALT $0

T6
I@0x00000000
SUB $2, $0, $7
SUB $6, $7, $0
LOAD $4, $4, $4
ADDI $5, $1, 0x4
ADD $7, $3, $0
ADD $6, $2, $2
STORE $2, $2, 0x3
SUB $2, $6, $2
ADD $5, $3, $1
HALT $0

T7
I@0x00000000
ADDI $4, $1, 0x4
SUBI $5, $2, 0x2
LOAD $7, $7, $2
SUB $3, $2, $5
HALT $0

T8
I@0x00000000
STORE $3, $7, $4
ADDI $2, $1, 0x3
LOAD $3, $0, $1
ADDI $3, $3, 0x2
HALT $0

T9
I@0x00000000
SUBI $1, $1, 0x1
ADDI $7, $4, 0x1
STORE $7, $0, 0x1
STORE $4, $2, $5
ADDI $1, $1, 0x4
STORE $6, $1, $6
LOAD $1, $1, $3
HALT $0

D@0x0
0x3
0x3
0x1
0x9
0xA
0x8
0x3
