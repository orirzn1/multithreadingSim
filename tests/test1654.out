
---- Blocked MT Simulation ----

Register file thread id 0:
	R0 = 0x0	R1 = 0xFFFFFFFE	R2 = 0x0	R3 = 0xFFFFFFFD	R4 = 0xFFFFFFFF	R5 = 0xFFFFFFFB	R6 = 0xFFFFFFFC	R7 = 0x0
Register file thread id 1:
	R0 = 0xFFFFFFFC	R1 = 0x3	R2 = 0xFFFFFFFF	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 2:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x4
Register file thread id 3:
	R0 = 0x0	R1 = 0x0	R2 = 0x1	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 4:
	R0 = 0x3	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0xFFFFFFFE	R6 = 0x0	R7 = 0x6
Register file thread id 5:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 6:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0xFFFFFFFE	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 7:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x2	R4 = 0x4	R5 = 0xFFFFFFFE	R6 = 0x0	R7 = 0xFFFFFFFE
Register file thread id 8:
	R0 = 0x0	R1 = 0x0	R2 = 0x3	R3 = 0x2	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 9:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x1
Blocked MT CPI for this program 7.458333

-----Finegrained MT Simulation -----

Register file thread id 0:
	R0 = 0x0	R1 = 0xFFFFFFFE	R2 = 0x3	R3 = 0xFFFFFFFD	R4 = 0xFFFFFFFF	R5 = 0xFFFFFFFB	R6 = 0xFFFFFFFC	R7 = 0x0
Register file thread id 1:
	R0 = 0xFFFFFFFC	R1 = 0x0	R2 = 0xFFFFFFFF	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 2:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x4
Register file thread id 3:
	R0 = 0x0	R1 = 0x0	R2 = 0x1	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 4:
	R0 = 0x3	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0xFFFFFFFE	R6 = 0x0	R7 = 0x6
Register file thread id 5:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 6:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 7:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x2	R4 = 0x4	R5 = 0xFFFFFFFE	R6 = 0x0	R7 = 0x0
Register file thread id 8:
	R0 = 0x0	R1 = 0x0	R2 = 0x3	R3 = 0x2	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 9:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x1
Finegrained Multithreading CPI for this program 1.166667

